Sources/PackageRegistryTool/PackageRegistryTool+Publish.swift

```diff
@@ -133,7 +140,15 @@ extension SwiftPackageRegistryTool {
                     callbackQueue: .sharedConcurrent,
                     completion: callback
                 )
-            }
+            }*/
+            let publishConfiguration = RegistryClient.PublishConfiguration(
+                metadata: .init(location: [.archive, .request]), // FIXME
+                signing: .init(
+                    required: self.signingIdentity != nil || self.privateKeyPath != nil,
```

Comment:
If either `privateKeyPath` or `certificateChainPaths` is set and non-empty, then the other must also be set.